packages/expo/cli/api/getProject.ts

```diff
@@ -0,0 +1,31 @@
+import { CommandError } from '../utils/errors';
+import { fetchAsync } from './rest/client';
+import { ensureLoggedInAsync } from './user/actions';
+
+interface Project {
+  accountId: string;
+  createdAt: string;
+  fullName: string;
+  id: string;
+  latestPublishedTime: null | unknown;
+  latestReleaseId: null | unknown;
+  latestRevisionId: null | unknown;
+  originalFullName: string;
+  packageUsername: string;
+  pageViewCount: null | unknown;
```

Comment:
Do we still send this down? Probably should remove it. Same with viewTrendScore.

Reply:
this is based on a project I pulled last night, the exact fixture and ID are in the test if you want to see.

Reply:
I'll take care of this on the server side: https://linear.app/expo/issue/ENG-4175/remove-unused-project-fields-like-pageviewcount-and-viewtrendscore